index.html

```diff
@@ -0,0 +1,134 @@
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="intro">
+        <p class="hello">"Hello! Zamalek fans"</p>
+        <h1>Election event for the most titled club in the 20th century.</h1>
+        <p class="header-description">
+          A general invitation to attend the Zamalek club elections for a new presidential term,
+          election festival , listen to candidates depates.
+        </p>
+        <p class="date">2022.11.10(THU) ~ 11(FRI)</p>
+        <p class="address">@ Cairo Nasser city zamalek club</p>
+      </section>
+      <section id="program" class="program">
+        <h2 class="program-title">Main program</h2>
+        <div class="program-cards">
+          <div class="card">
+            <img src="./images/teacher.png" alt="lecture-icon" />
+            <h3 class="card-title">Depates</h3>
+            <p>
+              Listen to candidates election programs about the next 4 years in the club.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/board.png" alt="exhibition-icon" />
+            <h3 class="card-title">Election zone</h3>
+            <p>Discover your election zone</p>
+          </div>
+          <div class="card">
+            <img src="./images/text.png" alt="forum-icon" />
+            <h3 class="card-title">Discuss</h3>
+            <p>Have time to share your thoughts, and think about your choise</p>
+          </div>
+          <div class="card">
+            <img src="./images/pen.png" alt="workshops-icon" />
+            <h3 class="card-title">Elect</h3>
+            <p>
+              Go to your election zone , wait for your turn to make your choise.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/net.png" alt="connect-icon" />
+            <h3 class="card-title">Results</h3>
+            <p>
+              gathering in the meeting room to announce the the results.
+            </p>
+          </div>
+        </div>
+        <div class="join-now">
+          <a class="save-spot" href="#">Save your spot for the Summit</a>
+          <a class="full-program" href="#">See the whole program</a>
+        </div>
+      </section>
+      <section class="candidates">
+        <h2>Candidates</h2>
+        <span></span>
+        <div class="candidate-cards"></div>
+        <div class="more-candidate">
+          <a href="#">MORE</a>
+        </div>
+      </section>
+      <section>
+        <div class="about-logo hide-mobile">
+          <h3>Partners</h3>
+          <div id="our-partners" class="partners">
+            <img src="./images/partner1.png" alt="" />
+            <img src="./images/partner2.png" alt="" />
+            <img src="./images/partner3.png" alt="" />
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="hide-mobile gray">
+      <a class="logo-home" href="./index.html">
+        <img src="./images/icons/logo.png" alt="logo"/></a>
```

Comment:
- [ Optional] The footer in the home page has a light silver background color (or you can make it white) and the gap between the logo and the text is narrower in the design. Besides there is no gap on the left and the bottom for the footer in the design.
#### Design
![image](https://user-images.githubusercontent.com/44057863/169667844-22783ca8-c4ef-4f55-84c9-45d944a7908a.png)
#### Impl
![image](https://user-images.githubusercontent.com/44057863/169667856-386d7c6e-b69d-480b-8a3b-dfbd3c0d0be2.png)